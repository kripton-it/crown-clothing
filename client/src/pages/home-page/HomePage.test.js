import React from "react";
import { shallow } from "enzyme";

import HomePage from "./HomePage";

it("expect to render HomePage page", () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
