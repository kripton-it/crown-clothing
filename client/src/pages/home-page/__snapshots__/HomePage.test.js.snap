// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render HomePage page 1`] = `ShallowWrapper {}`;
